homestead-prov-cassandra.root/usr/share/clearwater/cassandra-schemas/homestead_provisioning.sh

```diff
@@ -26,16 +26,21 @@ done
 
 CQLSH="/usr/share/clearwater/bin/run-in-signaling-namespace cqlsh"
 
+rc=0
+
 if [[ ! -e /var/lib/cassandra/data/homestead_provisioning ]] || \
    [[ $cassandra_hostname != "127.0.0.1" ]];
 then
   # replication_str is set up by
   # /usr/share/clearwater/cassandra-schemas/replication_string.sh
-  echo "CREATE KEYSPACE IF NOT EXISTS homestead_provisioning WITH REPLICATION = $replication_str;
-USE homestead_provisioning;
-CREATE TABLE IF NOT EXISTS implicit_registration_sets (id uuid PRIMARY KEY, dummy text) WITH COMPACT STORAGE AND read_repair_chance = 1.0;
-CREATE TABLE IF NOT EXISTS service_profiles (id uuid PRIMARY KEY, irs text, initialfiltercriteria text) WITH COMPACT STORAGE AND read_repair_chance = 1.0;
-CREATE TABLE IF NOT EXISTS public (public_id text PRIMARY KEY, publicidentity text, service_profile text) WITH COMPACT STORAGE AND read_repair_chance = 1.0;
-CREATE TABLE IF NOT EXISTS private (private_id text PRIMARY KEY, digest_ha1 text, realm text, plaintext_password text)
-WITH COMPACT STORAGE AND read_repair_chance = 1.0;" | $CQLSH
+  $CQLSH -e "CREATE KEYSPACE IF NOT EXISTS homestead_provisioning WITH REPLICATION = $replication_str;
+             USE homestead_provisioning;
+             CREATE TABLE IF NOT EXISTS implicit_registration_sets (id uuid PRIMARY KEY, dummy text) WITH COMPACT STORAGE AND read_repair_chance = 1.0;
+             CREATE TABLE IF NOT EXISTS service_profiles (id uuid PRIMARY KEY, irs text, initialfiltercriteria text) WITH COMPACT STORAGE AND read_repair_chance = 1.0;
+             CREATE TABLE IF NOT EXISTS public (public_id text PRIMARY KEY, publicidentity text, service_profile text) WITH COMPACT STORAGE AND read_repair_chance = 1.0;
+             CREATE TABLE IF NOT EXISTS private (private_id text PRIMARY KEY, digest_ha1 text, realm text, plaintext_password text)
```

Comment:
(Picky and you can ignore if you want)
Can this last line match the others? (so put "WITH COMPACT STORAGE.." on the line above.